plugins/inputs/logfile/logfile_test.go

```diff
@@ -394,8 +395,12 @@ func makeTempFile(t *testing.T, prefix string) *os.File {
 // getLogSrc returns a LogSrc from the given LogFile, and the channel for output.
 // Verifies 1 and only 1 LogSrc is discovered.
 func getLogSrc(t *testing.T, logFile *LogFile) (*logs.LogSrc, chan logs.LogEvent) {
+	start := time.Now()
 	logSources := logFile.FindLogSrc()
-	require.Equal(t, 1, len(logSources))
+	duration := time.Since(start)
+	// LogFile.FindLogSrc() should not block.
+	require.Less(t, duration, time.Millisecond*100)
```

Comment:
nit: why did we pick 100 milliseconds and isn't possible to make this a constant?